api/inc/rpc.h

```diff
@@ -29,10 +30,6 @@
 /* FIXME This is a dummy implementation. */
 #define UVISOR_BOX_RPC_MAX_INCOMING(max_num_incoming_rpc)
```

Comment:
Exposing this dummy API seems wrong to me, because

1. The max incoming PRC message count is fixed for all boxes at the moment.
2. This API is unbalanced, I'd expect to be able to define the `MAX_OUTGOING` message as well.

For the sake of future compatibility I'd like to add a `UVISOR_BOX_RPC_MAX_OUTGOING` macro as well.